cookbooks/aws-parallelcluster-config/templates/default/directory_service/sssd.conf.erb

```diff
@@ -0,0 +1,38 @@
+[domain/default]
+debug_level = 0x1f0
+id_provider = ldap
+cache_credentials = True
+ldap_schema = AD
+ldap_uri = <%=  node['cluster']["directory_service"]["domain_addr"] %> # TODO: use ldaps
```

Comment:
is the TODO at the end of this line taken care of now?

Reply:
No, I believe Simple AD doesn't support ldaps?

Reply:
we discussed this offline. it was removed because the TODO was for the integration test and not on the cookbook side